tools/docker/iroha-all-in-one/README.md

```diff
@@ -0,0 +1,29 @@
+# Iroha All in One Image
+
+An all in one Iroha + Postgres image that uses about 0.5 GB of RAM and one vCPU worth of hardware resources idling.
+This makes it much easier to pull up for tests that are about functionality rather than running a production grade ledger.
```

Comment:
I don't think that those information should be here.
1. I assume You didn't mean `RAM`?
2. Good docker instances are using as less memory as possible, so we don't add information about its, because it can change in future.
3. Instead of first line I think there should be information: what it is, what it is for, what it contains, etc.
4. I don't think that the second line is necessarily - user would see alternatives of getting docker container and choose one of them.

Reply:
Thank you for pointing out the potential way to improve!
I have updated the `README.md` file according to what we discussed today.